### PR TITLE
Cleanup scalability test configs

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -3514,6 +3515,7 @@
       "--extract=ci/latest",
       "--gcp-master-image=gci",
       "--gcp-node-image=gci",
+      "--gcp-node-size=g1-small",
```
Comment (shyamjvs) [inline jobs/config.json]: Maybe we can keep this (given that it saves us ~2x cost)?

Comment (wojtek-t) [reply jobs/config.json]: I don't understand. Keep what?

Comment (shyamjvs) [reply jobs/config.json]: Keep using g1-small nodes.

Comment (wojtek-t) [reply jobs/config.json]: I still don't understand - I want to use them that's why I added that.

Comment (shyamjvs) [reply jobs/config.json]: Oh sorry.. that's correct. I don't know what's wrong (I guess with my github UI) that I see the opposite diff.

**Pull request merged**